scripts/list-workspaces-for-docs.js

```diff
@@ -31,22 +32,54 @@ async function main(args) {
 
   const backendFeatureReports = [];
 
+  // Get `CODEOWNERS` entries
+  const codeownersPath = resolve(rootPath, '.github', 'CODEOWNERS');
+  const codeOwnerEntries = await codeowners.loadOwners(codeownersPath);
+
   // Get workspaces
   const workspaces = (await fs.readdir(workspacePath, { withFileTypes: true }))
     .filter(w => w.isDirectory() && !EXCLUDED_WORKSPACES.includes(w.name))
     .map(w => w.name);
 
   // Loop through workspaces
   for (const workspace of workspaces) {
+    // Find the owner by looking up the workspace in the `CODEOWNERS` file
+    const owners = codeOwnerEntries
+      .filter(c => c.pattern === `/workspaces/${workspace}`)
+      .map(o => o.owners)
+      .flat();
+    const filteredOwners = owners.filter(
+      o => o !== '@backstage/community-plugins-maintainers',
+    );
+    filteredOwners.forEach((owner, index) => {
+      if (owner === '@backstage/sda-se-reviewers') {
+        filteredOwners[
+          index
+        ] = `[@backstage/sda-se-reviewers](https://github.com/orgs/backstage/teams/sda-se-reviewers)`;
```

Comment:
Wdyt about something like:

```suggestion
      if (owner.includes('/') {
        const org = owner.substring(1, owner.indexOf('/'));
        const team = owner.substring(owner.indexOf('/') + 1);
        filteredOwners[
          index
        ] = `[${owner}](https://github.com/orgs/${org}/teams/${team})`;
```

Reply:
Left this in place intentionally as I don't think using groups makes sense for the Community Plugins repo. Based on this it didn't make sense to optimize for a single entry.